CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_jira v0.9.0
+
+## 🚨 Breaking Changes 🚨
+- PR [#66](https://github.com/fivetran/dbt_jira/pull/66) incorporates the following updates:
+  - Flipped column aliases `sum_close_time_seconds` and `sum_current_open_seconds` of intermediate model `int_jira__user_metrics.sql`.
+  - This ensures that downstream model `jira__user_enhanced.sql` calculates columns `avg_age_currently_open_seconds` and `avg_close_time_seconds` correctly.
```

Comment:
@fivetran-joemarkiewicz Is this what you were looking for?

Reply:
Hey @fivetran-catfritz I was thinking more of adding the link to each individual bullet.
```suggestion
## 🚨 Breaking Changes 🚨
 - Flipped column aliases `sum_close_time_seconds` and `sum_current_open_seconds` of intermediate model `int_jira__user_metrics.sql`. ([#66](https://github.com/fivetran/dbt_jira/pull/66))
 - This ensures that downstream model `jira__user_enhanced.sql` calculates columns `avg_age_currently_open_seconds` and `avg_close_time_seconds` correctly. ([#66](https://github.com/fivetran/dbt_jira/pull/66))
```

Reply:
This way once my breaking changes from PR #63 are included as well we just have one grain of bullets that list the corresponding PR.

Reply:
Ahh got it, thanks!